src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -1,6 +1,6 @@
 package mate.academy.service;
 
-public class AuthenticationService {
+public class AuthenticationService extends UserService {
```

Comment:
Same here: inheritance is not the case, because for sure we cannot say that `AuthenticationService` is `UserService`.